app/src/main/java/org/greenstand/android/TreeTracker/models/StepCounter.kt

```diff
@@ -41,6 +46,10 @@ class StepCounter(
         sensorManager.unregisterListener(stepCountEventListener)
     }
 
+    fun snapshotAbsoluteStepCountOnTreeCapture() {

```

Comment:
Added this function so that the variable absoluteStepCountOnTreeCapture  is modified only via this function and only read directly.